tests/endpoints/test_onshore_inventories_timeseries.py

```diff
@@ -22,7 +22,9 @@ def test_timeseries_returns_3_weeks_of_data(self):
             .to_df()
         )
 
-        assert len(df) == 3
+        # This used to be 3, but the result from API has changed to contain 4.
+        # TODO: Investigate why. This is to fix pipeline issues.
```

Comment:
I think your fix is correct and we should remove this comment. With the changes to OnshoreInventories weeks, this used to return Thursday to Thursday weeks 5-11, 12-18, and 19-23. Now this would return Monday - Sunday weeks 2-8, 9-15, 16-22, 23-23